app/Models/Event.php

```diff
@@ -45,7 +49,11 @@ public function getTrainDurationAttribute(): int {
     }
 
     public function getIsPrideAttribute(): bool {
-        $event_name_lowercase = strtolower($this->name);
-        return Str::contains($event_name_lowercase, ['csd', 'pride']);    
+        $eventNameLowercase = strtolower($this->name);
+        return Str::contains($eventNameLowercase, ['csd', 'pride']);
+    }
+
+    public function approvedBy(): HasOne {
+        return $this->hasOne(User::class, 'id', 'approved_by');
```

Comment:
unused and always-null function?

Reply:
Yes, but I kinda need it in an upcoming feature. Either I remove it now and re-add it tomorrow, or we leave it here. 😅